prometheus/prometheus.yml

```
@@ -0,0 +1,24 @@
+    static_configs:
+      - targets: ['host.docker.internal:19006']
+
+  - job_name: 'zipkin'
+    scrape_interval: 5s
+    metrics_path: '/prometheus'
+    static_configs:
+      - targets: ['zipkin:9411']
+
+  - job_name: 'redis'
+    static_configs:
+      - targets: ['redis:6379']
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)